src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -455,9 +453,7 @@ private static void NewMethod(out Construct obj1, out Construct obj2, out Constr
             await TestAsync(
 @"class Program { static void Main ( string [ ] args ) { [| (int a, int b) x = (1, 2); |]  System . Console . WriteLine ( x.a ); } } " + TestResources.NetFX.ValueTuple.tuplelib_cs,
 @"class Program { static void Main ( string [ ] args ) { (int a, int b) x = {|Rename:NewMethod|}(); System.Console.WriteLine(x.a); } private static (int a, int b) NewMethod() { return (1, 2); } }" + TestResources.NetFX.ValueTuple.tuplelib_cs,
-index: 0,
-parseOptions: TestOptions.Regular,
-withScriptOption: true);
+index: 0);
```

Comment:
index: 0 not needed either.  but you don't need to spin a new review.  Just keep in mind for the future.